docs/MAVLINKX.md

```diff
@@ -0,0 +1,15 @@
+# mLRS Documentation: MavlinkX #
+
+([back to main page](../README.md))
+
+MavlinkX is a technology which weeds out deficiencies of the original MAVLink protocol and makes the over-the-air MAVLink communication significantly more robust, reducing packet losses to the minimum. Furthermore, it can provide per-message compression of the MAVLink data stream.
+
+A major deficiancy of the MAVLink protocol is that, in the presence of data loss, it is prone to miss MAVLink packets which are actually valid, i.e., which are correctly and fully recieved and thus provide valid data, but are missed in the parsing of the stream (for a deeper discussion of such topics see [here](https://github.com/mavlink/mavlink/issues/1347)).
```

Comment:
How about having a parser heading, like done for 'exclude' and 'ortho'

Slight rework:

'A major deficiency of the MAVLink protocol is that in the presence of data loss, it is prone to miss MAVLink packets which are actually valid. The packets are correctly and fully received and could provide valid data, but are missed in the parsing of the stream (for a deeper discussion of such topics see [here](https://github.com/mavlink/mavlink/issues/1347)).'

Reply:
GREAT. done.